python/nano/src/bigdl/nano/deps/neural_compressor/inc_api_2.py

```diff
@@ -93,6 +97,7 @@ def _quantize(
     from neural_compressor.conf.pythonic_config import Config
     from neural_compressor.conf.config import QuantConf
     from neural_compressor.data import DataLoader
+    from neural_compressor.experimental.common import Metric as NCMetric
```

Comment:
why not `from neural_compressor.metric import Metric`

Reply:
This import is useless now, will delete this line.